core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/pluginloader/PluginManagerService.java

```diff
@@ -74,77 +76,76 @@
 	private ConfigurationManager configurationManager;
 	@Reference
 	private Configuration di;
-
-	@Reference
-	private CommandLineArgumentInfo cmaInfo;
-
 	@Reference
 	private Environment environment;
 
 	private Properties config;
 
-	/**
-	 * The root directory of the project. Defaults to the working directory.
-	 */
-	private Path projectDir;
+	private Path workingDir;
+
+	private boolean loaded = false;
 
 	@PostConstruct
 	private void setup() {
+		this.workingDir = this.environment.getWorkingDirectory();
+		loadAndCheckConfig();
+	}
+
+	private void loadAndCheckConfig() {
 		this.config = this.configurationManager.getConfigurationWithDefaults(CONFIG_NAME);
-		this.projectDir = this.environment.getWorkingDirectory();
+		checkProperty(PROPERTY_PLUGINS);
+		checkProperty(PROPERTY_PLUGIN_DIR);
+	}
+
+	private void checkProperty(String key) {
+		if (this.config.getProperty(key) == null)
```

Comment:
never null because of default config

Reply:
It can, when the default is wrong, so it is still good to Test it.